xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -224,19 +224,19 @@ ASS_Image* CDVDSubtitlesLibass::RenderImage(
   CSingleLock lock(m_section);
   if (!m_renderer || !m_track)
   {
-    CLog::Log(LOGERROR, "{} - ASS renderer/ASS track not initialized.", __FUNCTION__);
-    return NULL;
+    CLog::Log(LOGERROR, "{} - ASS renderer/ASS track not initialized.", __func__);
```

Comment:
here was used __FUNCTION__ because return the class+function name
where instead __func__ return only the function name

Reply:
will change it back, I was told `__func__` was normally advised

Reply:
from what i understand _`_FUNCTION__` come from C instead `__func__` is from C++ this last one is preferible but not return the class name, then you have to add it manually on the text string

Reply:
I "c" :)
Changed back to `__FUNCTION__` a global cleanup can be done later if so desired